include/eve/detail/function/simd/x86/make.hpp

```diff
@@ -32,48 +34,55 @@ namespace eve::detail
                   , "[eve::make] - Invalid number of arguments"
                   );
 
-    constexpr auto c = categorize<wide<T,S>>();
-
-          if constexpr( c == category::float64x2) return _mm_setr_pd(static_cast<T>(vs)...);
-    else  if constexpr( c == category::float32x4)
+    if constexpr(has_plain_translation<T>)
     {
-      return [&]<std::size_t... N>(std::index_sequence<N...> const&)
-      {
-        return _mm_setr_ps(vs..., (N ? 0:0)...);
-      }(std::make_index_sequence<4 - sizeof...(vs)>());
+      return make(as<wide<translate_t<T>, S>>{}, translate(vs)...);
```

Comment:
I don't like that we have to do so much work in custom ways. I'd seriosly consider migrating make first.